product_docs/docs/biganimal/release/getting_started/identity_provider/okta.mdx

```diff
@@ -0,0 +1,64 @@
+
+      | Name                               | Value          | Notes                                                                                                                                                                                               |
+      |------------------------------------|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+      | &lt;assertion_path>/name           | user.name      | The name field defaults to the user’s email. Alternatively, you can set the value as the following Okta expression to combine the first and last names: user.firstName + " " + user.lastName |
+      | &lt;assertion_path>/emailaddress   | user.email     |                                                                                                                                                                                                    |
+      | &lt;assertion_path>/nameidentifier | user.userId    | Optional                                                                                                                                                                                           |
+      | &lt;assertion_path>/surname        | user.lastName  |                                                                                                                                                                                                    |
+      | &lt;assertion_path>/givenname      | user.firstName |                                                                                                                                                                                                    |
+   
+   
+      Where &lt;assertion_path> is http://schemas.xmlsoap.org/ws/2005/05/identity/claims.
+
+   1. On the **Assignments** tab on the Applications page, select **Assign** to assign people or groups to the newly created application. If you need to sign into BigAnimal, be sure to assign yourself.
+   1.  On the **Sign On** tab, select **View SAML setup instructions** to open a tab with instructions for your application. 
+       1. Copy from the Identity Provider Single Sign-on URL from step 1 of the SAML setup instructions.
+       1. Select **Download certificate** in step 3 of the instructions.
+
+   1. In BigAnimal, on the **Setup Config** tab on the Set Up Identity Provider page: 
+      1. Paste the Identity Provider Single Sign-on URL you copied from Okta into the **Single Sign-On URL** field. 
+      1. For **Identity Provider Signature Certificate**, upload the certificate downloaded from Okta. 
+      1. Select the appropriate method for **Request Binding**. Okta supports HTTP POST and HTTP Redirect. 
+      1. For **Response Signature Algorithm**, select **rsa-sha256** .
```

Comment:
Okta supports both rsa-sha256 and rsh-sha1

Reply:
Incorporated